src/main/java/frc/robot/subsystems/elevator/ElevatorConstants.java

```diff
@@ -18,6 +21,10 @@ public class ElevatorConstants { // TODO: check real values
     public static final MutableMeasure<Distance> MAX_HEIGHT =
             Units.Meters.of(0).mutableCopy(); // [m]
 
+    public static final MutableMeasure<Angle> SERVO_OPEN = Units.Degrees.of(0).mutableCopy(); // [m]
+    public static final MutableMeasure<Angle> SERVO_CLOSE =
```

Comment:
Names should be a bit higher-level and represent their real meanings (e.g. "locked position").
Also, you don't need to write the unit in a comment -- the whole point about using measures is that you can use whatever unit you like. But anyway, the "meters" written in the comment doesn't match the `Degrees` object type...

Reply:
f6e729483a4a045c56b162619e86bccb6f372272